habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -1255,7 +1269,122 @@ def set_receptacle_in_scene(self, ep_info):
                 np.min(bounds, axis=0), np.max(bounds, axis=0)
             )
         return receps
-   
+
+    def set_desired_receptacle_in_scene(self, ep_info, receptacle_name):
```

Comment:
Maybe we could merge this function with `set_receptacles_in_scene`, they function basically the same.